README.md

```diff
@@ -4,14 +4,34 @@ Next.js HOC to integrate NProgress inside your app.
 
 This is configured to run only after a delay of (default) 300ms. This means if the page change takes too long it will render the progress bar, but if it's fast enough it will avoid rendering it.
 
-## Usage
-
-Install it
+## Installation
 
 ```bash
 yarn add next-nprogress
 ```
 
+## Usage
+
+### Component
+
+Import it inside your `pages/_app.js`;
+
+```js
+import NProgress from "next-nprogress/component";
+```
+
+Wrap the page content in your [custom App container](https://nextjs.org/docs#custom-%3Capp%3E) in it:
+
+```jsx
+<NProgress>
+  {pageContent}
+</NProgress>
```

Comment:
Could you make it work without children? just render `<NProgress />` and it should work instead of wrapping the pageContent